sci-libs/pytorch/pytorch-1.10.0.ebuild

```diff
@@ -0,0 +1,328 @@
+	rmdir third_party/onnx || die
+	ln -sv "${WORKDIR}"/onnx-29e7aa7048809784465d06e897f043a4600642b2 third_party/onnx || die
+	rmdir third_party/onnx-tensorrt || die
+	ln -sv "${WORKDIR}"/onnx-tensorrt-c153211418a7c57ce071d9ce2a41f8d1c85a878f third_party/onnx-tensorrt || die
+	rmdir third_party/onnx-tensorrt/third_party/onnx || die
+	ln -sv "${WORKDIR}"/onnx-765f5ee823a67a866f4bd28a9860e81f3c811ce8 third_party/onnx-tensorrt/third_party/onnx || die
+	rmdir third_party/onnx/third_party/benchmark || die
+	ln -sv "${WORKDIR}"/benchmark-e776aa0275e293707b6a0901e0e8d8a8a3679508 third_party/onnx/third_party/benchmark || die
+	rmdir third_party/psimd || die
+	ln -sv "${WORKDIR}"/psimd-072586a71b55b7f8c584153d223e95687148a900 third_party/psimd || die
+	rmdir third_party/pthreadpool || die
+	ln -sv "${WORKDIR}"/pthreadpool-a134dd5d4cee80cce15db81a72e7f929d71dd413 third_party/pthreadpool || die
+	rmdir third_party/python-peachpy || die
+	ln -sv "${WORKDIR}"/PeachPy-07d8fde8ac45d7705129475c0f94ed8925b93473 third_party/python-peachpy || die
+	rmdir third_party/QNNPACK || die
+	ln -sv "${WORKDIR}"/QNNPACK-7d2a4e9931a82adc3814275b6219a03e24e36b4c third_party/QNNPACK || die
+	rmdir third_party/sleef || die
+	ln -sv "${WORKDIR}"/sleef-e0a003ee838b75d11763aa9c3ef17bf71a725bff third_party/sleef || die
+	rmdir third_party/tensorpipe || die
+	ln -sv "${WORKDIR}"/tensorpipe-d2aa3485e8229c98891dfd604b514a39d45a5c99 third_party/tensorpipe || die
+	rmdir third_party/tensorpipe/third_party/googletest || die
+	ln -sv "${WORKDIR}"/googletest-2fe3bd994b3189899d93f1d5a881e725e046fdc2 third_party/tensorpipe/third_party/googletest || die
+	rmdir third_party/tensorpipe/third_party/libnop || die
+	ln -sv "${WORKDIR}"/libnop-aa95422ea8c409e3f078d2ee7708a5f59a8b9fa2 third_party/tensorpipe/third_party/libnop || die
+	rmdir third_party/tensorpipe/third_party/libuv || die
+	ln -sv "${WORKDIR}"/libuv-48e04275332f5753427d21a52f17ec6206451f2c third_party/tensorpipe/third_party/libuv || die
+	rmdir third_party/XNNPACK || die
+	ln -sv "${WORKDIR}"/XNNPACK-79cd5f9e18ad0925ac9a050b00ea5a36230072db third_party/XNNPACK || die
+	rmdir third_party/kineto || die
+	ln -sv "${WORKDIR}"/kineto-879a203d9bf554e95541679ddad6e0326f272dc1 third_party/kineto || die
+	rmdir third_party/breakpad || die
+	ln -sv "${WORKDIR}"/breakpad-7d188f679d4ae0a5bd06408a3047d69ef8eef848 third_party/breakpad || die
+	rmdir third_party/breakpad/src/third_party/lss || die
+	ln -sv "${WORKDIR}"/linux-syscall-support-e1e7b0ad8ee99a875b272c8e33e308472e897660 third_party/breakpad/src/third_party/lss || die
+
+	if use cuda; then
+		cd third_party/nccl/nccl || die
+		eapply "${FILESDIR}"/${PN}-1.6.0-nccl-nvccflags.patch
+		cuda_src_prepare
+		export CUDAHOSTCXX=$(cuda_gccdir)/g++
+	fi
+
+	if use rocm; then
+		#Allow escaping sandbox
+		addread /dev/kfd
+		addread /dev/dri
+		addwrite /dev/kfd
+		addwrite /dev/dri
+
+		ebegin "HIPifying cuda sources"
+		tools/amd_build/build_amd.py
```

Comment:
This calls python? In that case you should run it with an enabled python version using `${EPYTHON}`.

And we also need to `|| die` here